packages/commands/README.md

```diff
@@ -16,14 +58,6 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
-### CommandMenu
-
-Undocumented declaration.
-
-### privateApis
-
-Undocumented declaration.
-
 ### store
 
 Store definition for the commands namespace.
```

Comment:
It would be great to also add an example of using the Command Store. Another option would be to add some references of components using this store.

I would suggest adding something like:

----

```
import { store as commandsStore } from '@wordpress/commands';
import { useSelect, useDispatch } from '@wordpress/data';
...
const { open: openCommandCenter } = useDispatch( commandsStore );
```

Check how this Commands Store is being used in components like [`site-hub`](https://github.com/WordPress/gutenberg/blob/999b49c5ba9e07c0991e5b0cbfa2e4ac952fc738/packages/edit-site/src/components/site-hub/index.js#L23) or [`document-actions`](https://github.com/WordPress/gutenberg/blob/999b49c5ba9e07c0991e5b0cbfa2e4ac952fc738/packages/edit-post/src/components/header/document-actions/index.js#L14).

Reply:
Great example and links, I have added both.